infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/Warnings.java

```diff
@@ -52,5 +48,9 @@
       COMMAND_IS_CONFIGURED_IN_PLUGIN_WITH_MULTIPLY_CONTAINERS_WARNING_MESSAGE_FMT =
           "There are configured commands for plugin '%s' that has multiply containers. Commands will be configured to be run in first container";
 
+  public static final int VOLUME_MOUNT_SUBPATH_IS_NOT_PREFIXED_WARNING_CODE = 4108;
```

Comment:
Not sure that we should provide such a warning. Maybe we should throw an exception during runtime start instead. Like https://github.com/sleshchenko/che/commit/36e7bb9b0e31c2dd159385320fb15a4506f5f659.